### PR TITLE
Added PHP 8.3 Support

### PR DESCRIPTION
**Pull request by fulopattila122**
Hi There!

Besides enabling PHP 8.3 in the `composer.json` file, I've also added the supported PHP versions to the CI matrix to see if it actually works with the given versions.

See a [sample run](https://github.com/fulopattila122/mimey/actions/runs/6972910356) (passes) on my fork:

image

### CODE REVIEW

**Comment by fulopattila122** [comment]
Hi @ricardoboss, do you see a chance that it'll be merged?
If not, I can understand, no problem, but could you please drop me a line here so that I know that it won't be merged? :bow:

Thank you!

**Comment by ricardoboss** [comment]
Hi @fulopattila122! I'm reviewing this PR now as I didn't get notified until you commented...

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -8,11 +8,21 @@ on:
...
-    name: Run Unit Tests
+    name: Run Unit Tests PHP ${{ matrix.php }}
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: [ '8.1', '8.2', '8.3' ]
     steps:
     - uses: actions/checkout@v2
 
+    -  name: Installing PHP
```
Comment (ricardoboss) [inline .github/workflows/tests.yml]: "Installing" should be "Install" and I would include the version in the name, so:

```
- name: Install PHP ${{ matrix.php }}

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -8,11 +8,21 @@ on:
 
 jobs:
   test:
-    name: Run Unit Tests
+    name: Run Unit Tests PHP ${{ matrix.php }}
```
Comment (ricardoboss) [inline .github/workflows/tests.yml]: Drop the "PHP" from here, I'd say

**Review by ricardoboss — CHANGES_REQUESTED** [review]
Looks good! Just some small nitpicks.

File: .github/workflows/tests.yml (hunk 1 above)
Comment (fulopattila122) [reply .github/workflows/tests.yml]: OK, changes arriving shortly

File: .github/workflows/tests.yml (hunk 2 above)
Comment (fulopattila122) [reply .github/workflows/tests.yml]: ✅

**Review by ricardoboss — APPROVED** [review]

**Comment by fulopattila122** [comment]
Thank you!

**Pull request merged**